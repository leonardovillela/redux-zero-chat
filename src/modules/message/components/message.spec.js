import React from 'react';
import { shallow } from 'enzyme';

import Message from './message';

describe('Message component unit', () => {
  it('Should render component with correct structure', () => {
    const wrapper = shallow(<Message
      message={[]}
      createMessage={ jest.fn() }
      setMessageContent={ jest.fn() }
      messageContent=""
    />);

    expect(wrapper).toMatchSnapshot();
  });
});
