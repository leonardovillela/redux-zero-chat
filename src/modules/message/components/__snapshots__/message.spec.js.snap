// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component unit Should render component with correct structure 1`] = `
<div
  className="vh-100"
>
  <MessageList
    className="w-100 h-75"
    messages={Array []}
  />
  <MessageForm
    className="w-100 h-25"
    createMessage={[Function]}
    messageContent=""
    setMessageContent={[Function]}
  />
</div>
`;
